src/Microsoft.TemplateSearch.TemplateDiscovery/LocalFileSourcePackProvider.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.TemplateSearch.TemplateDiscovery.Nuget;
+using Microsoft.TemplateSearch.TemplateDiscovery.PackProviders;
+
+namespace Microsoft.TemplateSearch.TemplateDiscovery
+{
+    // Ad-hoc for test purposes (for now).
+    // Not referenced by any live code, but could becone the basis for a local source scraper implementation
+    public class LocalFileSourcePackProvider : IPackProvider
+    {
+        private static readonly string PathToPacks = @"";
+
+        public IEnumerable<IPackInfo> CandidatePacks
+        {
+            get
+            {
+                List<IPackInfo> packs = new List<IPackInfo>();
+
+                foreach (string packFileName in Directory.EnumerateFiles(PathToPacks))
+                {
+                    NugetPackInfo packInfo = new NugetPackInfo()
+                    {
+                        VersionedPackageIdentity = Path.GetFileName(packFileName),
+                        Id = Path.GetFileName(packFileName),
+                        Version = "0.0",
```

Comment:
Most of the stuff in this block feels off. The format of the filenames that we'll generally find is `{packageId}.{semanticVersion}.nupkg`. While I know this class isn't used yet, we should either not check it in, move it to tests (if it's only used for tests), or get it to be proper.